Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -250,7 +250,10 @@ def __call__(self, x):
         return y.abs().sum()
```

Comment:
I think here we forget to apply the weight to `b`

Reply:
Very good spot!